index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+</head>
+<body>
+
+<!--Start navbar-sec-sec-->
+    <nav class="navbar-sec">
+        <div class="welcome">Welcome</div>
+        <div class="icon-menu"> <img src="img/menu-icon.svg" alt=""></div>
+    </nav>
+  <!--End navbar-sec-sec-->
+
+      <!--Start Hero-->
+    <section class="hero-sec">
+        <header>
+            <h1>Hey there. I'm David <br>
+                <span>I'm a software developer</span></h1>
+        </header>
+        <div class="heading-text">
+            <p> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="icon-row">
+            <img class="svg-image" src="img/twitter.svg" alt="Twitter Logo">
+            <img class="svg-image" src="img/Linkedin-icon.svg" alt="LinkedIn Logo">
+            <img class="svg-image" src="img/medium.svg" alt="Medium Logo">
+            <img class="svg-image" src="img/github.svg" alt="Github Logo">
+            <img class="svg-image" src="img/angellist.svg" alt="Angellist Logo">
+        </div>
```

Comment:
- [ ] Please change this to an unordered list `ul` where each `li` is a social media link `a` with an icon that corresponds that. 😁 

The purpose of this is that when a user clicks on a social media icon it should take them to your profile on that social media platform.

Reply:
This has been done